roles/blackbox_exporter/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- include: "{{ deployment_method }}_deployment.yml"
+
+- name: prepare firewalld white list
```

Comment:
put firewalld task at the end of the main.yml,   and be consistent with other roles.